src/Modal.js

```diff
@@ -267,20 +275,27 @@ const Modal = React.createClass({
       );
     }
 
+    const modalProps = Object.assign(
```

Comment:
instead of the whitelist you can do something like `elementProps = _.omit(this.props, Object.keys(Modal.PropTypes))`